index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/style.css">
+  <title>Keep Lists</title>
+</head>
+
+<body>
+  <header>
+    <h1>Keep Lists</h1>
+  </header>
```

Comment:
```suggestion
  <header>
    <h1>Outra coisa</h1>
  </header>
```
Gostei mais do nome 'Outra coisa'. O que vc acha?

Reply:
kkk, acho que essa mudança vai ter que ser repassada para a equipe de produto revisar 😎